NuGet.config

```diff
@@ -16,6 +16,7 @@
     <add key="dotnet31-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3.1-transport/nuget/v3/index.json" />
     <!-- Used for the Rich Navigation indexing task -->
     <add key="richnav" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-buildservices/nuget/v3/index.json" />
+    <add key="vssdk" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json" />
```

Comment:
Are the bits you need available in another feed @JamesNK❔ `restore` gets slower every time we embiggen this collection

Reply:
They aren't. VS feeds should be temporary until the VS packages that Roslyn references move out of preview.